metrics/influxdb.go

```diff
@@ -0,0 +1,52 @@
+package metrics
+
+import (
+	"time"
+
+	gomet "github.com/rcrowley/go-metrics"
+	"github.com/vrischmann/go-metrics-influxdb"
+)
+
+// Influx :
+type Influx struct {
+	Addr, Username, Password string
+	Counter                  map[string]gomet.Counter
+	Histogram                map[string]gomet.Histogram
+}
+
+// New :
+func (influx Influx) New() Influx {
+	c := make(map[string]gomet.Counter)
+	h := make(map[string]gomet.Histogram)
+	c[Requests] = gomet.NewCounter()
+	gomet.Register(Requests, c[Requests])
+	c[Successes] = gomet.NewCounter()
+	gomet.Register(Successes, c[Successes])
+	s := gomet.NewExpDecaySample(1028, 0.015)
```

Comment:
i just use the example from https://github.com/rcrowley/go-metrics
here is a paper describe what's it means, but i haven't study my self
[http://dimacs.rutgers.edu/~graham/pubs/papers/fwddecay.pdf](http://dimacs.rutgers.edu/~graham/pubs/papers/fwddecay.pdf)